### DISCUSSION TITLE
No dependency on distributed storage; single server nodes are autonomous... Big question then

### DISCUSSION THREAD

**Discussion by timothywcrane** [prometheus/prometheus#9405] [Q&A] [answered]
- [x] "No dependency on distributed storage; single server nodes are autonomous "

I understand that it does not depend upon it, but will it cause obvious problems?
I am...
1. on Kubernetes Cluster

2. with a [SeaweedFS distributed FS](https://github.com/chrislusf/seaweedfs), 

3. both on the same nationally distributed (4 locations), cluster (4 nodes total) , 

4. carpeted wall to wall with Distributed Storage via non-CSI [SeaweedFS Operator](https://github.com/seaweedfs/seaweedfs-operator) 
on a physical Kube of 1Mother & 3Siblings

**Comment by amye** [comment]
Hi there, 
Suggest that you reframe (4) in a more productive way, we do have a [code of conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md) that applies to all projects, and the comment as it's written isn't the best wording.

**Reply by timothywcrane** [reply]
What are Kubernetes cluster nodes to be called now on github. I was being productive. The official terms are still master and slave... Cleaned up.

**Comment by roidelapluie** [comment, marked as answer]
Prometheus requires Posix-compliant filesystems. While many distributed filesystems claim that they have this, in practice we have seen many issues in the past from users using NFS or other distributed storage solutions. In general, we do not support them. We recommend you to use a local filesystem.

 https://prometheus.io/docs/prometheus/latest/storage/

**Reply by timothywcrane** [reply]
Thank you.
